openpgp/read.go

```diff
@@ -343,10 +342,8 @@ func (scr *signatureCheckReader) Read(buf []byte) (n int, err error) {
 			if scr.md.SignatureError == nil && scr.md.Signature.SigExpired(scr.config.Now()) {
 				scr.md.SignatureError = errors.ErrSignatureExpired
 			}
-		} else if scr.md.SignatureV3, ok = p.(*packet.SignatureV3); ok {
-			scr.md.SignatureError = scr.md.SignedBy.PublicKey.VerifySignatureV3(scr.h, scr.md.SignatureV3)
 		} else {
-			scr.md.SignatureError = errors.StructuralError("LiteralData not followed by Signature")
+			scr.md.SignatureError = errors.StructuralError("LiteralData not followed by valid signature packet")
```

Comment:
Also here, if the signature packet was invalid I think `packets.Next()` will have errored already

Reply:
You mean remove this `else` case altogether? I only changed the message (before, an extraneous non signature packet was the scope of this error, and now, an extraneous non signature packet, or a v3 signature packet, now foreign to this library).

Reply:
I meant the same thing as here: https://github.com/ProtonMail/crypto/pull/55#discussion_r446990201
I.e. this error can't be a v3 signature, so the old error message makes more sense imo

Reply:
@twiss do you mean this is unreachable code or that this would be better as "LiteralData not followed by Signature"?